recipe/meta.yaml

```diff
@@ -1,46 +1,59 @@
 {% set name = "gguf" %}
-{% set version = "0.6.0" %}
+{% set version = "0.9.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gguf-{{ version }}.tar.gz
-  sha256: b2e22eaba2a106c1ad148186a14ad9f29c429351686fe9d7ce16df39a0607e65
+  sha256: f5e70987e15b19c545f6a9f7533b4033fb306330eb1f3c5c95fd28b14c3fd33b
 
 build:
+  entry_points:
+    - gguf-convert-endian = scripts:gguf_convert_endian_entrypoint
+    - gguf-dump = scripts:gguf_dump_entrypoint
+    - gguf-set-metadata = scripts:gguf_set_metadata_entrypoint
+    - gguf-new-metadata = scripts:gguf_new_metadata_entrypoint
   noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
-  entry_points:
-    - gguf-convert-endian = "scripts:gguf_convert_endian_entrypoint"
-    - gguf-dump = "scripts:gguf_dump_entrypoint"
-    - gguf-set-metadata = "scripts:gguf_set_metadata_entrypoint"
 
 requirements:
   host:
     - python >=3.8
+    - poetry-core >=1.0.0
     - pip
-    - poetry-core
   run:
     - python >=3.8
     - numpy >=1.17
+    - tqdm >=4.27
+    - sentencepiece >=0.2.0
```

Comment:
Can you add a comment to mention that upstream is missing this dependency? I found it in https://github.com/ggerganov/llama.cpp/blob/gguf-v0.9.1/gguf-py/gguf/vocab.py#L10. You night as well create an issue upstream to tell them they are missing a dep in their package metadata.

Reply:
@JeanChristopheMorinPerso Good call out - it's actually defined really weirdly in upstream - the tell users to install all the requirements here: https://github.com/ggerganov/llama.cpp/tree/master/requirements but for the gguf-py pyproject.toml it's missing. Will note and file upstream

Reply:
@JeanChristopheMorinPerso Added, opened a PR in upstream here: https://github.com/ggerganov/llama.cpp/pull/9107